lib/dom_global_attributes.ml

```diff
@@ -0,0 +1,29 @@
+include Dom_dsl_core.Prop
+
+let accessKey = string "accessKey"
+
+let className = string "className" (* substitute for "class" *)
+
+let contentEditable = bool "contentEditable"
+
+let contextMenu = string "contextMenu"
```

Comment:
This attr is obsolete and will be removed, I wonder if we should just not add it https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contextmenu

Reply:
Done in aa8a2e5